projects/demo/src/modules/components/editor/embed/youtube/editor-embed-youtube.component.html

```diff
@@ -0,0 +1,35 @@
+<tui-doc-page
+    header="Editor - Youtube"
+    package="ADDON-EDITOR"
+    type="components"
+>
+    <ng-template pageTab>
```

Comment:
Do we actually need to create tab if it is the single tab on the page ? 🤔 

<img width="982" alt="Screenshot 2023-02-08 at 13 03 44" src="https://user-images.githubusercontent.com/35179038/217498558-2f2fa3c4-7cb4-43a2-94af-93267ea6b80c.png">

Reply:
<img width="809" alt="image" src="https://user-images.githubusercontent.com/12021443/217506564-41255d0d-2b59-4494-9546-ac271a7f88d2.png">